modules/ossm-security-auth-policy.adoc

```diff
@@ -0,0 +1,180 @@
+////
+Module included in the following assemblies:
+-service_mesh/v2x/ossm-security.adoc
+////
+
+[id="ossm-vs-istio_{context}"]
+= Role Based Access Control (RBAC)
+
+Role-based access control (RBAC) objects determine whether a user or service is allowed to perform a given action within a project. You can define mesh-, namespace-, and workload-wide access control for your workloads in the mesh. 
+
+To configure RBAC, create an `AuthorizationPolicy` resource in the namespace for which you are configuring access. If you are configuring mesh-wide access, use `istio-system`. The namespace must match your `metadata.namespace` field.
+
+For example, with RBAC, you can create policies that:
+
+* Configure intra-project communication.
+* Allow or deny full access to all workloads in the default namespace.
+* Configure ingress gateway access.
+* Require a token for access,
+
+An authorization policy includes a selector, an action, and a list of rules:
+
+* The `selector` field specifies the target of the policy.
+* The `action` field specifies whether to allow or deny the request.
+* The `rules` specify when to trigger the action.
```

Comment:
@neal-timpe line 24 should be "The `rules` field specifies" instead of "specify"